src/components/select/ui/ValueContainer/useOverflow.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useRef, useState } from "react";
+import { SelectProps } from "../../types";
+
+export const useOverflow = (value: SelectProps["value"]) => {
+  const [isOverflow, setIsOverflow] = useState(false);
```

Comment:
that's a good point but what about a case where it could be inside some elastic component or window resizing? And it will reduce clientWidth as example

Reply:
Good. I thought about it but forgot during implementation

Reply:
Done.